test/err.js

```diff
@@ -27,3 +28,16 @@ test('type error', function (t) {
         '{ [SyntaxError: whoa] message: \'whoa\', \'a-b\': 5 }'
     ].join(', ') + ' ]');
 });
+
+test('type AssertionError', function (t) {
+    t.plan(3);
+    var err = 'woot', obj2str = Object.prototype.toString;
```

Comment:
Please use one `var` per variable declaration, so that a debugger can step through both initializations separately.